README.md

```diff
@@ -1,155 +1,156 @@
-
-```ts
-
-// config.default.ts
-export default {
-  AWS_SECRETS_MANAGER_NAME: 'production/myapp/config',
-  AWS_SECRETS_MANAGER_REGION: 'us-west-2',
-  AWS_SECRETS_MANAGER_TIMEOUT: 2000
-  API_ENDPOINT: 'https://api.somecompany.com'
-};
-```
-
-The default region is `us-east-1` and the default connection timeout is `5000`ms.
-
-Config Dug will warn if it detects invalid config values. Invalid values include:
-
-- undefined
-- null
-- the string 'undefined'
-- an empty string
-
-This package uses the [aws-sdk](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/) internally. Refer to their documentation for information about [authentication](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html), configuring a default [region](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html) and configuring [access control for AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/auth-and-access.html).
-
-## Advanced
-
-### Customizing Config Loading
-
-If you want to load config files from a directory other than the project root you can import the `loadConfig` function and use it directly.
-
-```ts
-import { loadConfig } from 'config-dug';
-
-loadConfig('config');
-```
-
-This will import your config files from the `config` directory. The path you specify must be relative to your project root.
-
-### Debugging
-
-`config-dug` uses the [debug](https://github.com/visionmedia/debug) library. To print debug messages for `config-dug` set `DEBUG=config-dug`.
-
-## Contributing
-
-### Running Tests
-
-1. Fork this repo
-1. Clone the forked repo
-1. Install dependencies: `npm install` OR `npm i`
-1. Run tests: `npm run test`
-
-## Publishing
-
-1. Update the version in `package.json`
-1. Add a `CHANGELOG` entry
-1. Commit your changes
-1. Run `npm pack` to see what will be published then delete the `.tgz` file that was created
-1. Run `npm publish`
-1. Create a release on GitHub. Use the version as the tag and release name. For example for version `1.0.0` the tag and release name would be `v1.0.0`.
-
-## Credits
-
-This project was inspired by [config3](https://github.com/focusaurus/config3) and [config4](https://github.com/autolotto/config4).
+# Config Dug
+
+[![Build status](https://github.com/neofinancial/config-dug/workflows/CI/badge.svg)](https://github.com/neofinancial/config-dug/actions)
+[![codecov](https://codecov.io/gh/neofinancial/config-dug/branch/master/graph/badge.svg)](https://codecov.io/gh/neofinancial/config-dug)
+![TypeScript 3.7.2](https://img.shields.io/badge/TypeScript-3.7.2-brightgreen.svg)
+[![coverage](https://coverage.neotools.ca/api/badge/master/4C8hhW7NzS4PKdA9C3NqNM)](https://coverage.neotools.ca/coverage/neofinancial/config-dug)

```

Comment:
I'm not sure why it re-added the entire READ.ME file but this is the only line that I added.

Reply:
I would try reverting your changes to this file and the reapplying them. It's weird that it's showing the whole file as changed.